packages/core/src/types.ts

```diff
@@ -112,18 +112,37 @@ export type AccountCenterPosition =
   | 'bottomLeft'
   | 'topLeft'
 
-export type AccountCenterOptions = {
-  desktop?: {
-    position?: AccountCenterPosition
-    enabled?: AccountCenter['enabled']
-  }
-}
-
 export type AccountCenter = {
-  enabled: boolean
-  position: AccountCenterPosition
-  expanded: boolean
+    enabled: boolean
+    position?: AccountCenterPosition
+    expanded?: boolean
+    minimal?: boolean
 }
+  
+export type AccountCenterOptions = {
+    desktop: Omit<AccountCenter, 'expanded'>
+    mobile: Omit<AccountCenter, 'expanded'>
+}
+  
+// export type AccountCenterOptions = {
```

Comment:
Remove commented out code when you have a min

Reply:
Done